### ISSUE TITLE
Box observation_space high bound wrongly set by wrap_deepmind

### ISSUE THREAD

**Issue by alexpalms** [hill-a/stable-baselines#829]
wrap_deepming function, depending on user inputs, could call first `ScaledFloatFrame` and then `FrameStack`, in this case the upper bound of the box observation space is set to 255 by `FrameStack`, while it should be 1.0

A possible solution could be to call `ScaledFloatFrame` at the end, even if this means getting rid of LazyFrames.

**Comment by araffin** [comment]
Hello,
That's true. In practice, this wrapper is not used and the scaling is done during the pre-processing.
We may remove that wrapper as it is also now present in gym.

**Comment by alexpalms** [comment]
Yep, I see normalization is performed in the observation_input method indeed, but if someone uses the wrapper with scale=True kwarg that's an issue. Thanks for the feedback.
